evi-next-js-function-calling/src/components/ClientComponent.tsx

```diff
@@ -82,15 +84,11 @@ const handleToolCall: ToolCallHandler = async (
   }
 };
 
-export default function ClientComponent({
-  accessToken,
-}: {
-  accessToken: string;
-}) {
+export default function ClientComponent({ accessToken }: { accessToken: string }) {
   return (
     <VoiceProvider
-      configId={process.env.NEXT_PUBLIC_HUME_CONFIG_ID}
-      auth={{ type: "accessToken", value: accessToken }}
+      configId={process.env.HUME_CONFIG_ID}
```

Comment:
same issue here, also it's ok for this environment variable to be prefixed with `NEXT_PUBLIC_` since it is not sensitive